.github/workflows/ci.yml

```diff
@@ -0,0 +1,374 @@
+            outputFilter: xcpretty
+          - xcode: "Xcode_14.1"
+            runsOn: macOS-12
+            name: "macOS 12, SPM 5.7.1 Test"
+            outputFilter: xcpretty
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install Firewalk
+        run: brew install alamofire/alamofire/firewalk || brew upgrade alamofire/alamofire/firewalk xcbeautify && firewalk &
+      - name: Test SPM
+        run: swift test -c debug | ${{ matrix.outputFilter }}
+  Linux:
+    name: Linux
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - image: swift:5.8-focal
+          - image: swift:5.8-jammy
+          - image: swift:5.8-centos7
+          - image: swift:5.8-amazonlinux2
+          - image: swift:5.8-rhel-ubi9
+          - image: swift:5.9-focal
+          - image: swift:5.9-jammy
+          - image: swift:5.9-centos7
+          - image: swift:5.9-amazonlinux2
+          - image: swift:5.9-rhel-ubi9
+          - image: swiftlang/swift:nightly-focal
+          - image: swiftlang/swift:nightly-jammy
+          - image: swiftlang/swift:nightly-amazonlinux2
+    container:
+      image: ${{ matrix.image }}
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@v4
+      - name: ${{ matrix.image }}
+        run: swift build --build-tests -c debug
+  Android:
+    name: Android
+    uses: hggz/swift-android-sdk/.github/workflows/sdks.yml@ci
+    strategy:
+      fail-fast: false
+    with:
+      target-repo: ${{ github.repository }}
+      checkout-hash: ${{ github.sha }}
+  Windows:
+    name: ${{ matrix.name }}
+    runs-on: windows-latest
+    timeout-minutes: 10
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - branch: swift-5.8-release
+            tag: 5.8-RELEASE
+            name: Windows Swift 5.8
+          - branch: swift-5.9-release
+            tag: 5.9-RELEASE
+            name: Windows Swift 5.9
+          - branch: swift-5.10-release
+            tag: 5.10-RELEASE
+            name: Windows Swift 5.10
+    steps:
+      - name: Setup
+        uses: compnerd/gha-setup-swift@main
+        with:
+          branch: ${{ matrix.branch }}
+          tag: ${{ matrix.tag }}
+      - name: Clone
+        uses: actions/checkout@v4
+      - name: SPM ${{ matrix.name }}
+        run: |
+          swift build --build-tests -c debug -Xlinker /INCREMENTAL:NO -v
+  CodeQL:
+    name: Analyze with CodeQL
+    runs-on: macOS-13
+    env:
+      DEVELOPER_DIR: "/Applications/Xcode_14.3.1.app/Contents/Developer"
+    timeout-minutes: 10
+    steps:
+      - name: Clone
+        uses: actions/checkout@v4
+      - name: Initialize CodeQL
+        uses: github/codeql-action/init@v2
+        with:
+          languages: swift
+      - name: Build macOS
+        run: set -o pipefail && env NSUnbufferedIO=YES xcodebuild -project "Alamofire.xcodeproj" -scheme "Alamofire macOS" -destination "platform=macOS" clean build | xcpretty
```

Comment:
Don't we need to change the reference to `FSLineChart` instead of `Alamofire` here?

Reply:
Here is that7e2d0729154d8f8d2fd61ddef0062e11316ebfe0

But really I have no idea to make testing work.

We can delete that part if it is not doing anything helpful